src/com/rath/rathbot/cmd/disc/actions/BanCmd.java

```diff
@@ -84,4 +84,4 @@ public boolean executeCommand(final IMessage msg, final String[] tokens, final i
     
     return RBCommand.STOP_CMD_SEARCH;
   }
-}
+}
```

Comment:
The JDK compiler doesn't seem to care if an EOL character is absent, but some do. Just stick a new line at the end of this file.